Runtime/Client/LootLockerServerApi.cs

```diff
@@ -0,0 +1,541 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+using System;
+using System.Text;
+using LootLocker.LootLockerEnums;
+using UnityEditor;
+using LootLocker.Requests;
+
+namespace LootLocker.LootLockerEnums
+{
+    public enum LootLockerCallerRole { User, Admin, Player, Base };
+}
+
+namespace LootLocker
+{
+    public class LootLockerServerApi : MonoBehaviour
+    {
+        private static LootLockerServerApi _instance;
+        private const int MaxRetries = 3;
+        private int _tries;
+
+        public static LootLockerServerApi GetInstance()
+        {
+            if (_instance == null)
+            {
+                _instance = new GameObject("LootLockerServerApi").AddComponent<LootLockerServerApi>();
+
+                if (Application.isPlaying)
+                    DontDestroyOnLoad(_instance.gameObject);
+            }
+            return _instance;
+        }
+
+        private void Awake()
```

Comment:
This Awake()-fuinction can be removed.
GetInstance() above adds DontDestroyOnLoad() when it has created the new object, so no need to use Awake().

Reply:
Great, I didn't dare remove it as I didn't know why it was there in the first place ^^ Thank you

Reply:
Done